### PR TITLE
encode text before sending to Element#type to support unicode

### PR DESCRIPTION
**Pull request by EndangeredMassa**
Addresses: https://github.com/groupon-testium/testium/issues/124

### CODE REVIEW

File: src/element.coffee [hunk 1]
```diff
@@ -57,6 +58,8 @@ parseElement = (http, elementId) ->
   else
     null
 
+encodeUTF = (string) ->
+  unescape(encodeURIComponent(string))
```
Comment (jkrems) [inline src/element.coffee]: Any reason to use `unescape` instead of `decodeURIComponent` here?

Comment (jkrems) [inline src/element.coffee]: Maybe an example of the transformation + a link would be nice as a comment here. http://stackoverflow.com/a/619428/916619

Comment (EndangeredMassa) [inline src/element.coffee]: Yeah, this a great place for a comment. Will add.

**Comment by EndangeredMassa** [comment]
Added a comment to the UTF encoding method

**Comment by jkrems** [comment]
:shipit:

**Comment by EndangeredMassa** [comment]
published in [1.0.0](https://github.com/groupon-testium/webdriver-http-sync/releases/tag/v1.0.0)

**Comment by johan** [comment]
Nice that it worked!

**Pull request merged**